server/services/queue.js

```diff
@@ -3,38 +3,36 @@ import queueManager from '../managers/queue';
 import config from '../../config/env';
 import { Job } from 'kue';
 
-const ONE_MINUTE_MILLISECONDS = 60 * 1000;
-
 queueManager.process('page', config.pageQueueSize, processPageQueue);
 
 function bulkCreatePages(pageRequests) {
   if (!pageRequests) {
     return Promise.resolve();
   }
   const ticketPromises = pageRequests.map((page) => {
-    const ticket = page.ticket;
-    const user = page.user;
-    const deviceIndex = page.deviceIndex;
-    return createDelayedPage(ticket, user, deviceIndex);
+    const ticketId = page.ticketId;
+    const userId = page.userId;
+    const device = page.device;
+    const delay = page.delay;
+    const title = page.title;
+    return createDelayedPage(ticketId, userId, device, delay, title);
   });
 
   return Promise.all(ticketPromises);
 }
 
-function createDelayedPage(ticket, user, deviceIndex) {
+function createDelayedPage(ticketId, userId, device, delay, title) {
   const jobDetails = {
-    ticket,
-    user,
-    device: user.devices[deviceIndex],
-    title: ticket.metadata.title
+    ticketId,
+    userId,
+    device,
+    title: `${title} - ${device.name}`
```

Comment:
It's so that you don't have multiple requests with the same title, just usability if they use the kue front end